### PR TITLE
Carpool travel method selected

### PR DESCRIPTION
**Pull request by Amutha37**
## Overview of the Pull Request:
Counter button for carpool travel method on selection  : 

- This is child component PR and dependent on PR 86 and 87. 

- This button will pop up on the bottom of all the travel method buttons.
- This component will have a break line for mobile version only.
-  The component stay on the  left float for desktop version and centered in mobile version

Screen Shot 2022-04-05 at 3 16 14 pm
Screen Shot 2022-04-05 at 3 17 17 pm

## How Has This Been Tested?
- _Please describe the tests that you ran to verify your changes._
- _Provide instructions so we can reproduce. _
- _If necessary, please also list any relevant details for your test setup/configuration_

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [ ] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [ ] Is your pull request up-to-date with `main` branch?
- [ ] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [ ] Have you written instructions on how to test that your changes work as expected?
- [ ] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

**Comment by hqtan** [comment]
Hi @Amutha37 ,

I cannot see the `Counter` component used in the `TravelMethod` page.
You've created the `Counter` component  in `components/TravelMethodButtons/CarpoolCounter.jsx`, but haven't used in anywhere.

I've tried to test out the `CarpoolCounter` component in the preview deployment of the `TravelMethod` page to test your changes, but it does not appear when I click on the `Carpool` button.
https://council-emissions-calculator-spike-5j7m8gjp8-codeforaustralia.vercel.app/form/TravelMethod

It's possible that I've misunderstood what this PR is meant to do. From the description above, I had assumed that you were implementing the `CarpoolCounter` component.
But with no instructions provided on how to test this PR, and what I should expect to see, it's difficult to understand this PR's intentions.

Can you please provide those details?

File: pages/form/TravelMethod.jsx [hunk 1]
```diff
@@ -78,8 +78,21 @@ export default function Question4() {
                 Select the ways you generally travel to work.
               </Text>
             </Box>
-            <TravelMethodButtons handleTransportMode={handleTransportMode} />
-
+            <TravelMethodButtons handleTransportMode={handleTransportMode}  />
+{/* Collapse on carpool selection*/}
+<Collapse in={transportMode === "Carpool"}>
```
Comment (hqtan) [inline pages/form/TravelMethod.jsx:83]: please fix indentation of code here:
```suggestion
        {/* Collapse on carpool selection*/}
        <Collapse in={transportMode === "Carpool"}>
```

File: pages/form/TravelMethod.jsx [hunk 2]
```diff
@@ -78,8 +78,21 @@ export default function Question4() {
...
+            mb={5}
+            fontSize={[15, 17]}
+            px="20px"
+            py="12px"
+            color="#155724"
+            bg="#D4EDDA"
+          >
+           How many other people would you most likely to carpool with?
+          </Text>
+        </Collapse>
```
Comment (hqtan) [inline pages/form/TravelMethod.jsx:94]: Have you forgotten to add the `Counter` component here?
```suggestion
        </Collapse>
        <Counter handleMinus handlePlus count />
```

**Review by hqtan — CHANGES_REQUESTED** [review]

**Comment by amu3** [comment]
To answer  to your request for change , need to have a clear understanding
fron figma. Initially applied on click (carppol) the collapse on carpool.
but looking at figma. it show while carppol is selected with other mode the
counter component stay open.

If this is the case then I have to check the answers from from the object.
At any given time if Carpool and other mode selected the counter remain
open.

This is what I understand from figma.

At the moment the object to save the answers are not ready so I have
temporary commented it.

**Comment by amu3** [comment]
On Sun, 10 Apr 2022 at 9:12 am, hqtan ***@***.***> wrote:

> Hi @Amutha37 <https://github.com/Amutha37> ,
>
> I cannot see the Counter component used in the TravelMethod page.
> You've created the Counter component in
> components/TravelMethodButtons/CarpoolCounter.jsx, but haven't used in
> anywhere.
>
> I've tried to test out the CarpoolCounter component in the preview
> deployment of the TravelMethod page to test your changes, but it does not
> appear when I click on the Carpool button.
>
> https://council-emissions-calculator-spike-5j7m8gjp8-codeforaustralia.vercel.app/form/TravelMethod
>
> It's possible that I've misunderstood what this PR is meant to do. From
> the description above, I had assumed that you were implementing the
> CarpoolCounter component.
> But with no instructions provided on how to test this PR, and what I
> should expect to see, it's difficult to understand this PR's intentions.
>
> Can you please provide those details?
>
> — I have made it in a way has described on my PR.
>
     The child component has to be cleared and merged then the parent
component.

I have tested on my side locally uding show state for carpool collpase. It
> is just for testing purpose, once the object amswer is implemented I will
> need to correct this lines of code.
>
  PR # 82 and PR #86 has to be approved and merged . I will at the counter
component that missing.

**Review by OlaShabalina — APPROVED** [review]

**Pull request merged**